rerun_py/rerun_sdk/rerun/log/image.py

```diff
@@ -39,11 +39,11 @@ def log_image(
 
     Supported dtypes
     ----------------
-     - uint8: color components should be in 0-255 sRGB gamma space, except for alpha which should be in 0-255 linear
-    space.
-     - uint16: color components should be in 0-65535 sRGB gamma space, except for alpha which should be in 0-65535
-    linear space.
-     - float32, float64: all color components should be in 0-1 linear space.
+     - uint8, uint16, uint32, uint64: color components should be in 0-`max_uint` sRGB gamma space, except for alpha
+       which should be in 0-`max_uint` linear space.
+     - float16, float32, float64: all color components should be in 0-1 linear space.
+     - int8, int16, int32, int64: signed integers are cast into their unsigned counterpart without clipping (why will
+       typically induce artefacts).
```

Comment:
I don't think this is true.

We have some heuristics in `tensor_data_range_heuristic` which basically boils down to:
* if all the data is in the 0-255 range, then treat is the same as a uint8 im that range
* if not, normalize to the actual data range

This means that if you log values between -517 and +24, then -517 is treated as black and +24 as white. The alpha component is even more complicated.

Reply:
OK, this was based on some odd comment in `api_demo/main.py` (I believe). I've adjusted the wording.